### DISCUSSION TITLE
Add unstable rate as win condition to multiplayer

### DISCUSSION THREAD

**Discussion by nyannoying1337** [ppy/osu#15203] [Ideas]
What about adding Unstable Rate as Win Condition to multiplayer? Currently on Stable we have multiple Win Conditions which include Accuracy. ( Shown in screenshot ). If Lazer gets win conditions maybe adding Unstable Rate could be considered as one as i feel this would fit very well in a game like this.
![image](https://user-images.githubusercontent.com/17437302/138058843-e7f897f7-b094-47df-8282-58fc8cce1b1a.png)

**Comment by Nyanekoy** [comment]
That would be cool, i wonder if its possible to make the Hit Accuracy to be part of it as well. Like who hit all the circles closer to the middle? 🤔

**Comment by peppy** [comment]
Could we please not propagate UR out to more places? I'd like to rethink how it is calculated as a number first. It's very arbitrary and hard to understand right now. It would make more sense if it was in milliseconds, for instance, not milliseconds *100 or whatever.

**Comment by frenzibyte** [comment]
Is this a suggestion for osu!(stable)? this should be tracked in https://github.com/ppy/osu-stable-issues instead.

**Reply by nyannoying1337** [reply]
No

**Reply by frenzibyte** [reply]
That should've probably been pointed out as I couldn't tell from the screenshot and the fact that osu!lazer doesn't seem to even have win condition support yet.

**Reply by nyannoying1337** [reply]
Fair enough. i figured "like we have on stable" would have been enough to make this point out as a Lazer addition. i changed it a bit for clarification ( hope that works ). But as peppy and you pointed out Lazer doesn't have win conditions yet and Unstable Rate is supposed to be changed first. So maybe a consideration for later on then.
